extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamInfo.java

```diff
@@ -181,13 +181,16 @@ private static StreamInfo extractStreams(StreamInfo streamInfo, StreamExtractor
 
         // Either audio or video has to be available, otherwise we didn't get a stream
         // (since videoOnly are optional, they don't count).
-        if ((streamInfo.videoStreams.isEmpty()) && (streamInfo.audioStreams.isEmpty())) {
+        if (streamInfo.videoStreams.isEmpty()
+                && streamInfo.audioStreams.isEmpty()
+                && isNullOrEmpty(streamInfo.getHlsUrl())
+                && isNullOrEmpty(streamInfo.getDashMpdUrl())) {
```

Comment:
Mmmh, I think before only videoStreams and audioStreams were checked because in any other case the stream is a livestream. So NewPipe's player expected to need to use hls/dash urls only on livestreams. Does this change break that workflow, or is the check being done differently by NewPipe and so this is not an issue?